keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -123,9 +123,9 @@ For alpha, the feature gates will be used for testing purposes. When enabled, te
 disabled by default.
 
 For beta, the feature gates will be on by default, meaning core components will disallow use of in-tree cloud providers. This will act as a warning for users to migrate to external components. Users may
-choose to continue using the in-tree provider by explicitly disabling the feature gates. Beta is targeted for v1.26 with the caveat that a majority of our CI signal jobs across providers should have converted to use CCM by then.
+choose to continue using the in-tree provider by explicitly disabling the feature gates. Beta is targeted for v1.28 with the caveat that a majority of our CI signal jobs across providers should have converted to use CCM by then.
```

Comment:
PR to switch CI jobs https://github.com/kubernetes/kubernetes/pull/114557

Reply:
updated version # here